distribution/docs/src/main/resources/content/_whitelists/platform-whitelist.adoc

```diff
@@ -6,31 +6,38 @@
 :order: 02
 :summary: ${ddf-platform} whitelist.
 
-The following packages have been exported by the ${ddf-platform} application and are approved for use by third parties:
-
-* `ddf.action`
-* `ddf.security`
-* `ddf.security.assertion`
-* `ddf.security.common.audit`
-* `ddf.security.http`
-* `ddf.security.permission`
-* `ddf.security.policy.extension`
-* `ddf.security.principal`
-* `ddf.security.samlp`
-* `ddf.security.service`
-* `ddf.security.settings`
-* `ddf.security.sts.client.configuration`
-* `ddf.security.ws.policy`
-* `ddf.security.ws.proxy`
-* `org.codice.ddf.branding`
-* `org.codice.ddf.configuration`
-* `org.codice.ddf.configuration.admin`
-* `org.codice.ddf.configuration.migration`
-* `org.codice.ddf.configuration.persistence`
-* `org.codice.ddf.configuration.persistence.felix`
-* `org.codice.ddf.configuration.status`
-* `org.codice.ddf.notifications.store`
-* `org.codice.ddf.parser`
-* `org.codice.ddf.parser.xml`
-* `org.codice.ddf.platform.error.handler`
-* `org.codice.ddf.platform.util`
+The following classes have been exported by the ${ddf-platform} application and are approved for use by third parties:
+
+In package `ddf.action`
+
+* `Action`
+* `ActionProvider`
+* `ActionRegistry`
+
+In package `org.codice.ddf.branding`
+
+* `BrandingPlugin`
+* `BrandingRegistry`
+
+In package `org.codice.ddf.configuration`
+
+* `ConfigurationWatcher` Deprecated
+
+In package `org.codice.ddf.configuration.persistence`
+
+* `PersistenceStrategy`
+
+In package `org.codice.ddf.notifications.store`
+
+* `NotificationStore` Comment says it is "Internal Notification service"
```

Comment:
❓ This was only whitelisted in DDF 2.9 even though it's been around a lot longer than that so it looks like it was whitelisted by mistake.

Reply:
removed